internal/provisioner/kops_provisioner_cluster_installation.go

```diff
@@ -194,14 +194,43 @@ func (provisioner *KopsProvisioner) UpdateClusterInstallation(cluster *model.Clu
 
 	version := translateMattermostVersion(installation.Version)
 	if cr.Spec.Version == version {
-		logger.Infof("Cluster installation already on version %s", version)
+		logger.Debugf("Cluster installation already on version %s", version)
+	} else {
+		logger.Debugf("Cluster installation version updated from %s to %s", cr.Spec.Version, installation.Version)
+		cr.Spec.Version = version
 	}
-	cr.Spec.Version = version
 
 	if cr.Spec.Image == installation.Image {
-		logger.Infof("Cluster installation already on Image %s", installation.Image)
+		logger.Debugf("Cluster installation already on image %s", installation.Image)
+	} else {
+		logger.Debugf("Cluster installation image updated from %s to %s", cr.Spec.Image, installation.Image)
+		cr.Spec.Image = installation.Image
+	}
+
+	if cr.Spec.Size == installation.Size {
+		logger.Debugf("Cluster installation already on size %s", installation.Size)
+	} else {
+		// A few notes on installation sizing changes:
```

Comment:
I wish we could surface these notes about the current limitations somewhere more visible than this comment, perhaps in release notes?

Reply:
Yeah, totally agree that this needs to be in more places. I often feel like these types of comments can really be helpful especially when trying to work out a fix to something in a pinch. That said, good idea about surfacing it elsewhere.